pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -281,7 +281,7 @@ func getProxyEnv(cfg *config.Cluster) (map[string]string, error) {
 }
 
 func getSubnets(networkName string) ([]string, error) {
-	format := `{{range (index (index . "IPAM") "Config")}}{{index . "Subnet"}} {{end}}`
+	format := `{{index (index  .plugins  0).ipam.ranges 0 0 "subnet"}}`
```

Comment:
has this changed again in podman?
it seems that the new command works for me in 3.0.0 but not the older one

Reply:
we need to provide all the networks, this returns only the first subnet
```
sudo podman network inspect -f '{{index (index  .plugins 0 ).ipam.ranges 0 0 "subnet"}}' kind
fc00:8866:27d0:bd7e::/64
```

```
sudo podman network inspect -f json  kind
[
    {
        "cniVersion": "0.4.0",
        "name": "kind",
        "plugins": [
            {
                "bridge": "cni-podman4",
                "hairpinMode": true,
                "ipMasq": true,
                "ipam": {
                    "ranges": [
                        [
                            {
                                "subnet": "fc00:8866:27d0:bd7e::/64"
                            }
                        ],
                        [
                            {
                                "subnet": "10.89.2.0/24"
                            }
                        ]

```

Reply:
Yes but I don't know how, I'm not good at templates.

Reply:
```
sudo podman network inspect kind -f '{{ range (index (index (index (index . "plugins") 0 ) "ipam" ) "ranges")}}{{ index ( index . 0 ) "subnet" }} {{end}} '
fc00:8866:27d0:bd7e::/64 10.89.0.0/24
```

Reply:
@flobz can you update the PR with this template ^^^?